apps/web/src/routes/_content/courses/index.tsx

```diff
@@ -601,25 +49,27 @@ function CoursesExplorer() {
       title={t('courses.explorer.exploreCourses')}
       subtitle={t('courses.explorer.journey')}
       description={t('courses.explorer.pageDescription')}
-      paddingXClasses="px-2.5 md:px-4"
-      maxWidth="max-w-[1227px]"
+      paddingXClasses="px-0"
+      maxWidth="max-w-[3000px]"
     >
       {!isFetched && <Loader size={'s'} />}
 
-      {courses && <CoursesGallery courses={courses} />}
+      <div className="max-w-[1227px] mx-auto px-2.5">
+        {courses && <CoursesGallery courses={courses} />}
+      </div>
       <div className="border-t border-newGray-1 max-w-[1115px] w-full mx-auto"></div>
```

Comment:
![image](https://github.com/user-attachments/assets/87dd5068-2c28-4bd8-b669-8c92a0e556d6)
Shouldn't take whole page (same for the other one)